pkg/logproto/compat.go

```diff
@@ -214,3 +214,91 @@ func init() {
 	jsoniter.RegisterTypeEncoderFunc("logproto.LegacySample", SampleJsoniterEncode, func(unsafe.Pointer) bool { return false })
 	jsoniter.RegisterTypeDecoderFunc("logproto.LegacySample", SampleJsoniterDecode)
 }
+
+// Combine unique values from multiple LabelResponses into a single LabelResponse.
+// TODO(jordanrushing): Should we sort labels lexicographically before returning?
```

Comment:
@cyriltovena what is the API doing?

Reply:
Yeah I think it does sort lexicographically.